code/__HELPERS/colors.dm

```diff
@@ -64,3 +64,26 @@
 		final_color += copytext(color, digit, digit + 1)
 
 	return final_color
+
+
+/// returns HSV brightness 0 to 100 by color hex
+/proc/get_color_brightness_from_hex(A)
+	if(!A || length(A) != length_char(A))
+		return 0
+	var/R = hex2num(copytext(A, 2, 4))
+	var/G = hex2num(copytext(A, 4, 6))
+	var/B = hex2num(copytext(A, 6, 8))
+	return round(max(R, G, B)/2.55, 1)
```

Comment:
don't simplify the constant expression, it's confusing. do / 255 / 100

Reply:
`/ 255 / 100` is a lot more confusing imo as it is ambiguous if this means `(x / 255) / 100` or `x / (255 / 100)` which could either be `x/25500` or `x/2.55`. In the expression requested, this would be equivalent to `x/25500` and not `x/2.55`.